docs/modules/ROOT/pages/relay.adoc

```diff
@@ -160,6 +160,190 @@ All private keys are stored in the AWS Key Management Service. Keys are generate
 
 As for API secrets, these are only kept in memory during creation when they are sent to the client. After that, they are hashed and stored securely in AWS Cognito, which is used behind the scenes for authenticating Relayer requests. This makes API keys easy to rotate while preserving the same private key on the KMS.
 
+[[relay-api]]
+== Relay API Reference
+The Relay API exposes two endpoints `txs` and `sign`. The `txs` endpoint allows to send transactions to Ethereum blockchain and query their status. The `sign` endpoint allows to sign arbitrary data with relayer's private key.
+
+=== Authentication
+A JWT Token is requred to make requests to Relay API. For security reasons authentication uses https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol[SRP protocol] and it is not possbile to retrive a JWT Token via one HTTP request. What is why it is advised to use the https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-integrate-apps.html[Amazon Cognito User Pool SDK] to retreive a JWT token.
+
+
+For python users there is no official AWS SDK, but it is possible to retrive a JWT token using the https://github.com/capless/warrant#cognito-srp-utility[warrant] library.
```

Comment:
> For python users there is no official AWS SDK

I found https://aws.amazon.com/sdk-for-python/ that seems to contradict it. If what you meant is that there's no official AWS SDK _for getting a token_, let's clarify that!